src/Target.ts

```diff
@@ -149,39 +149,57 @@ export class Target {
         throw new Error('Could not merge changes for file: ' + sourceFile.filename + '. Merge not supported.');
     }
 
-    private _parseCode(filename: string, sourceCode: string): null | GeneratedFile {
-        let mode = 'write-always';
-        let permissions = '644';
-        const lines = sourceCode.split(/\n/g).filter((line) => {
-            if (line.indexOf('#FILENAME:') > -1) {
-                filename = line.split(/#FILENAME:/)[1];
-                if (filename.indexOf(':') > -1) {
-                    [filename, mode, permissions] = filename.split(/:/g);
-                }
+    private _parseCode(filename: string, sourceCode: string): GeneratedFile[] {
+        const files: GeneratedFile[] = [];
+        let currentFileContent: string[] = [];
+        let currentFilename = filename;
+        let currentMode = 'write-always';
+        let currentPermissions = '644';
 
-                if (!permissions) {
-                    permissions = '644';
-                }
+        const processCurrentFile = () => {
+            if (currentMode === 'skip') {
+                return null;
```

Comment:
looks like this just should return ? (E..g not null just ```return;``` )

Reply:
Fixed here fb2fdb49540007f5351dbfe3230a370a0591bcde